app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\Category;
+use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Session;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $data = Category::all();
+
+        return view('pages.categories.list-categories')->with(compact('data'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        if(Session::get('level') == 'Admin') {
```

Comment:
e check đoạn này với mục đích làm gì vậy Phong?

Reply:
dạ vì yêu cầu đề bài là chỉ có tài khoản admin mới được quyền thêm thể loại nên em để thế ạ, nếu đúng là tài khoản admin thì được dùng còn không thì phải đăng nhập lại.

Reply:
@DTP-IT  doạn này em đọc thêm phần Authen trong laravel để sau có cách làm chuyên nghiệp hơn nhé, chị có đưa vài cái link trong phần comment đó.  
E chỉ cần check ngay từ route như vậy : 

```
Route::group(['middleware' => 'auth.admin'], function (){
    Route::post('create',  'CategoryController@create')->name('category.create');
});
```

Reply:
@mvn-huyenle-hn dạ vâng, phần này bài sau em dùng Auth của Laravel ạ